src/rust/engine/src/nodes.rs

```diff
@@ -956,24 +961,38 @@ impl Task {
     params: Params,
     entry: Arc<rule_graph::Entry<Rule>>,
     generator: Value,
-  ) -> NodeResult<Value> {
```

Comment:
The doc comment on this function needs to be updated

Reply:
Done!